pom.xml

```diff
@@ -96,15 +96,15 @@
         <java.version>11</java.version>
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
-        <peppol.sbdh.version>2.2.0</peppol.sbdh.version>
-        <cxf.version>3.6.4</cxf.version>
-        <wss4j.version>2.4.1</wss4j.version>
-        <jakarta.xml.soap-api.version>1.4.2</jakarta.xml.soap-api.version>
-        <saaj-impl.version>1.5.3</saaj-impl.version>
-        <jakarta.xml.bind-api.version>2.3.3</jakarta.xml.bind-api.version>
-        <jaxb-impl-version>2.3.3</jaxb-impl-version>
-        <jakarta.annotation-api.version>1.3.5</jakarta.annotation-api.version>
-        <jakarta.xml.ws-api.version>2.3.3</jakarta.xml.ws-api.version>
+        <peppol.sbdh.version>2.2.1-SNAPSHOT</peppol.sbdh.version>
+        <cxf.version>4.0.5</cxf.version>
+        <wss4j.version>3.0.3</wss4j.version>
```

Comment:
Make sure that you do Not change version of other libraries than one relevant with this specific pull request. 
Any other library change require impact analysis and will be done through separate pull request. No big-bang change per commit.